generate/adapter.py

```diff
@@ -104,23 +117,9 @@ def main(
     model.load_state_dict(checkpoint)
     fabric.print(f"Time to load the model weights: {time.perf_counter() - t0:.02f} seconds.", file=sys.stderr)
 
-    model.eval()
     model = fabric.setup(model)
 
-    tokenizer = Tokenizer(checkpoint_dir)
```

Comment:
Curious what's the impact of moving these blocks of code up?

Reply:
No impact speed-wise. I originally was creating the kv cache inside `init_module` since I don't need to do it in the loop anymore but `empty_init` messed it up with fsdp so I went back to using `init_tensor`.

TLDR: this change is not necessary anymore, I could revert it if you prefer the previous order